src/middleware/navigation/check.condition.ts

```diff
@@ -5,15 +5,12 @@ import { HasSoldLandKey, IsSecureRegisterKey, EntityNameKey, EntityNumberKey } f
 import { PresenterKey } from '../../model/presenter.model';
 import { EntityKey } from '../../model/entity.model';
 import { BeneficialOwnerStatementKey } from '../../model/beneficial.owner.statement.model';
-import { BeneficialOwnerGovKey } from '../../model/beneficial.owner.gov.model';
-import { BeneficialOwnerIndividualKey } from '../../model/beneficial.owner.individual.model';
-import { BeneficialOwnerOtherKey } from '../../model/beneficial.owner.other.model';
-import { ManagingOfficerCorporateKey } from '../../model/managing.officer.corporate.model';
-import { ManagingOfficerKey } from '../../model/managing.officer.model';
 import { WhoIsRegisteringKey } from '../../model/who.is.making.filing.model';
 import { OverseasEntityDueDiligenceKey } from '../../model/overseas.entity.due.diligence.model';
 import { DueDiligenceKey } from '../../model/due.diligence.model';
 import { UpdateKey } from '../../model/update.type.model';
+import { checkBOsDetailsEntered } from '../../utils/application.data';
```

Comment:
Do both in one line
import { checkBOsDetailsEntered, checkMOsDetailsEntered } from '../../utils/application.data';
Surprised lint did not complain